src/stylesheets/reset.css

```diff
@@ -0,0 +1,46 @@
+html {
```

Comment:
This should be an scss file, not css. Otherwise we are recompiling it into css for no reason.